conans/model/requires.py

```diff
@@ -143,6 +143,13 @@ def __str__(self):
                                              self.visible)
         return "{}, Traits: {}".format(self.ref, traits)
 
+    def serialize(self):
+        return {attribute: getattr(self, attribute) for attribute in ("run", "libs", "skip",
```

Comment:
where is ``ref``?

Reply:
In the tests but did not stash the changes for this file 🥴